psci/PSCi.hs

```diff
@@ -59,6 +59,35 @@ import Parser (parseCommand)
 import Completion (completion)
 import Types
 
+-- | The name of the PSCI support module
+supportModuleName :: P.ModuleName
+supportModuleName = P.ModuleName [P.ProperName "$PSCI", P.ProperName "Support"]
+
+-- | Support module, contains code to evaluate terms
+supportModule :: P.Module
+supportModule = 
+  case P.parseModulesFromFiles id [("", code)] of
+    Right [(_, P.Module cs _ ds exps)] -> P.Module cs supportModuleName ds exps
+    _ -> error "Support module could not be parsed"
+  where
+  code :: String
+  code = unlines
```

Comment:
Haskell doesn't have `"""` multiline strings :wink:

It does have the `\` stuff, but that's not really any better, and strips linebreaks.